marlowe-dashboard-client/src/Dashboard/Types.purs

```diff
@@ -28,12 +29,14 @@ type State
     , menuOpen :: Boolean
     , screen :: Screen
     , cards :: Array Card
+    , status :: ContractStatus
```

Comment:
I think just `status` made more sense when the ContractHome and Play were separated (and maybe not even then... my bad). But I think this could be renamed to `filterBy`, or `contractFilter` or similar.

Reply:
Agreed. I'll leave it for now, but try out a different name in the next PR.